retrofit/src/main/java/retrofit/Endpoint.java

```diff
@@ -8,7 +8,10 @@
 public abstract class Endpoint {
   /** Create an endpoint with the provided {@code url}. */
   public static Endpoint createFixed(String url) {
-    checkNotNull(url, "url == null");
+    checkNotNull(url, "URL == null");
```

Comment:
This was lowercase to denote the method parameter name. The others referenced the uppercase acronym as a noun.